tests/InputFiles/Cce/CharacteristicExtract.yaml

```diff
@@ -24,73 +29,117 @@ ResourceInfo:
       AdamsBashforth:
-        Order: 3
+        Order: 3 # Going to higher order doesn't seem necessary for CCE
     StepChoosers:
-      - Constant: 0.5
+      - Constant: 0.5 # Don't take steps bigger than 0.5M
       - Increase:
           Factor: 2
       - ErrorControl(SwshVars):
           AbsoluteTolerance: 1e-8
           RelativeTolerance: 1e-6
+          # These factors control how much the time step is changed at once.
           MaxFactor: 2
           MinFactor: 0.25
+          # How close to the "perfect" time step we take. Since the "perfect"
+          # value assumes a linear system, we need some safety factor since our
+          # system is nonlinear, and also so that we reduce how often we retake
+          # time steps.
           SafetyFactor: 0.9
       - ErrorControl(CoordVars):
           AbsoluteTolerance: 1e-8
           RelativeTolerance: 1e-7
+          # These factors control how much the time step is changed at once.
           MaxFactor: 2
           MinFactor: 0.25
+          # How close to the "perfect" time step we take. Since the "perfect"
+          # value assumes a linear system, we need some safety factor since our
+          # system is nonlinear, and also so that we reduce how often we retake
+          # time steps.
           SafetyFactor: 0.9
 
+  # The number of angular modes used by the CCE evolution. This must be larger
+  # than ObservationLMax. We always use all of the m modes for the LMax since
+  # using fewer m modes causes aliasing-driven instabilities.
   LMax: 20
-  NumberOfRadialPoints: 12
+  # Probably don't need more than 15 radial grid points, but could increase
+  # up to ~20
+  NumberOfRadialPoints: 15
+  # The maximum ell we use for writing waveform output. While CCE can dump
+  # more, you should be cautious with higher modes since mode mixing, truncation
+  # error, and systematic numerical effects can have significant contamination
+  # in these modes.
   ObservationLMax: 8
 
   InitializeJ:
+    # To see what other J-initialization procedures are available, comment
+    # out this group of options and do, e.g. "Blah:" The code will print
+    # an error message with the available options and a help string.
+    # More details can be found at spectre-code.org.
     ConformalFactor:
       AngularCoordTolerance: 1e-13
-      MaxIterations: 1000
-      RequireConvergence: False
+      MaxIterations: 1000 # Do extra iterations in case we improve.
+      RequireConvergence: False # Often don't converge to 1e-13, but that's fine
       OptimizeL0Mode: True
       UseBetaIntegralEstimate: False
       ConformalFactorIterationHeuristic: SpinWeight1CoordPerturbation
       UseInputModes: False
       InputModes: []
 
-  StartTime: 0.0
-  EndTime: 1000.0
+  StartTime: Auto # Start at the first time in file
+  EndTime: Auto   # End at the last time in file
   BoundaryDataFilename: CceR0257.h5
+  # WARNING: The ExtractionRadius must be updated with the file name!
+  ExtractionRadius: 257.0
```

Comment:
For this, the code will actually print a warning if you try to specify the extraction radius when the file name also has the extraction radius. The recommended way is either
```yaml
  BoundaryDataFilename: CceR0257.h5
  ExtractionRadius: Auto
```
or
```yaml
  BoundaryDataFilename: NameWithoutRadius.h5
  ExtractionRadius: 257.0
```

I know this is a bit confusing, though.